README.md

```diff
@@ -1 +1,146 @@
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+> This is my first Microverse Project. It contains some things I am capable of doing as a software developer.
+
+**HELLO MICROVERSE PROJECT** is a responsive website that tests my web development skills. 
+
+## 🛠 Built With <a name="HTML and CSS"></a>
+
+ This is a Mobile First website.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.org/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://css.org/">CSS</a></li>
+  </ul>
+</details>
+
+
+- **Channge the content color**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser that supports html5 and css3
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/AbednegoKibitok/set-up-and-mobile-first.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+- Clone the code into your machine and run test in an editor of your choice.
+
+### Usage
+
+To run the project, execute the following command:
+
+- http://127.0.0.1:5500/index.html
+
+### Run tests
+
+To run tests, run the following command:
+
+- http://127.0.0.1:5500/index.html
+
+### Deployment
+
+You can deploy this project using:
+
+- Deploy it on a browser of your choice.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/AbednegoKibitok)
+- Twitter: [@twitterhandle](https://twitter.com/abedysongol)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abednego-kibitok-58717517a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Contact form]**
+- [ ] **[Desktop version of the website]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/AbednegoKibitok/set-up-and-mobile-first/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```

Comment:
- [ ] kindly as requested by the previous reviewer Please remove all these comments that begin with > as they were placed in this README file as a guide for you and should have been removed after editing this file